### DISCUSSION TITLE
GraphQL query returns "forbidden"

### DISCUSSION THREAD

**Discussion by soke1556** [requarks/wiki#3853] [Error / Bug Report]
Any query to users results in the error below. The api key is unrestricted and other queries and mutations work fine. The only one failing is this.

```
{
    "errors": [
        {
            "message": "Forbidden",
            "locations": [
                {
                    "line": 3,
                    "column": 5
                }
            ],
            "path": [
                "users",
                "list"
            ],
            "extensions": {
                "code": "INTERNAL_SERVER_ERROR"
            }
        }
    ],
    "data": {
        "users": {
            "list": null
        }
    }
}
```

**Comment by joshii-h** [comment]
Idk a lot, but the recent change could be the reason:
#3056

Hope it helped!

**Comment by soke1556** [comment]
Thanks @joshii-h but I believe it doesn't explain why some queries work and other do not. Right?

**Comment by soke1556** [comment]
Also, I'm on 2.5.191

**Comment by joshii-h** [comment]
Okay, what if, when u update it to the latest [Release](https://github.com/Requarks/wiki/releases), and do the changes, as mentioned under #3056 Prob it is resolved by the changes otherwise I'm probably the wrong person ._. 
But I hope that it will work afterwards!

**Comment by soke1556** [comment]
I'm afraid that if I update to 2.5.201 I won't be able to query anymore from other domains as I'm doing right now. Currently I only have restrictions for some queries, and not all of them.

**Comment by soke1556** [comment]
Can @NGPixel clarify this, please? Thanks in advance.

**Reply by TBG-FR** [reply]
See https://github.com/Requarks/wiki/discussions/3056#discussioncomment-541268
You'll have to edit the code while we wait for 3.0 (and I'll have to do the same)
